src/Analyzers/MSTest.Analyzers/Resources.resx

```diff
@@ -684,4 +684,13 @@ The type declaring these methods should also respect the following rules:
   <data name="IgnoreStringMethodReturnValueDescription" xml:space="preserve">
     <value>Methods like Contains, StartsWith, and EndsWith return boolean values that indicate whether the condition was met. Ignoring these return values is likely a mistake.</value>
   </data>
+  <data name="DoNotUseParallelizeAndDoNotParallelizeTogetherTitle" xml:space="preserve">
+    <value>Do not use both Parallelize and DoNotParallelize attributes</value>
```

Comment:
```suggestion
    <value>Do not use both 'Parallelize' and 'DoNotParallelize' attributes</value>
```

We should also add `Locked` to prevent localization of the attribute names.